src/web/lib/decidePalette.ts

```diff
@@ -349,6 +358,7 @@ const backgroundArticle = (format: Format): string => {
 	if (format.design === Design.Comment) return opinion[800];
 	if (format.design === Design.Editorial) return opinion[800];
 	if (format.theme === Special.SpecialReport) return specialReport[800]; // Note, check theme rather than design here
+	if (format.theme === Special.Labs) return neutral[86];
```

Comment:
```suggestion
	if (format.theme === Special.Labs) return neutral[97];
```
Doms new designs use this bg colour instead for higher contrast